runner.py

```diff
@@ -67,6 +70,18 @@ def run_agent(terminate_event, callback_event, name, team, index, module_name):
     callbacks = []
     name_dict = dict()
 
+    save_data = True
+    save_path = os.getcwd() + '\\training'
+    game_name = str(int(round(time.time() * 1000))) + '-' + str(random.randint(0, 1000))
```

Comment:
I don't think random part is necessary. Just seconds seems sufficient to me.  Chance of 2 people running at same second is slim.